examples/bluetooth/MIT-App-Inventor/ble.py

```diff
@@ -0,0 +1,52 @@
+# This code requires the micropython_dotstar and ble_advertising libraries
```

Comment:
This is obvious from the imports, not really needed to repeat?

Reply:
Thank you for looking into this request. I'm happy to remove that should you find this example worth including.